### PR TITLE
11.2.0 Changelog

### PR DESCRIPTION
**Pull request by ryanthemanuel**
https://github.com/cypress-io/cypress-documentation/pull/4869

### CODE REVIEW

**Review by elylucas — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

File: content/_changelogs/11.2.0.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## 11.2.0
+
+_Released 11/22/2022_
+
+**Features:**
+
+- Adds a new experimental flag, `experimentalRunAllSpecs`, for running multiple
```
Comment (emilyrohrbough) [inline content/_changelogs/11.2.0.md:7]:
```suggestion
- Re-implemented the 'Run All Specs' for End to End Testing for running multiple tests sequentially in open mode. This feature had been previously removed in Cypress 10.0. Enable 'Run All Specs' with the new experimental flag, `experimentalRunAllSpecs`. Please leave any feedback around 'Run All Specs'
  [here](https://github.com/cypress-io/cypress/discussions/21628). Your feedback
  will help us make product decisions around the future of this feature. Addresses
  [#24168](https://github.com/cypress-io/cypress/issues/24168).
```

File: content/_changelogs/11.2.0.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  tests sequentially in Open Mode for End to End testing. Addresses
+  [#24168](https://github.com/cypress-io/cypress/issues/24168) and
+  [this discussion](https://github.com/cypress-io/cypress/discussions/21628)
+- Added messaging in CI in run mode to direct people to set up recording to the
+  dashboard on failures if they are not already set up to do so. Addresses
+  [#24000](https://github.com/cypress-io/cypress/issues/24000)
+- Visually warn in the Command Log when cy.session() fails to restore a session
```
Comment (emilyrohrbough) [inline content/_changelogs/11.2.0.md:14]:
```suggestion
- Visually warn in the Command Log when `cy.session()` fails to restore a session
```
Can we add a link to the session API

File: content/_changelogs/11.2.0.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  to surface to users potential issues with their setup and validation.
+  Addresses [#24207](https://github.com/cypress-io/cypress/issues/24207)
+
+**Bugfixes:**
+
+- The session command no longer clears the page between setup and validation and
```
Comment (emilyrohrbough) [inline content/_changelogs/11.2.0.md:20]:
```suggestion
- The cy.session() command no longer clears the page between setup and validation and
```

File: content/_changelogs/11.2.0.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+_Released 11/22/2022_
+
+**Features:**
+
+- Adds a new experimental flag, `experimentalRunAllSpecs`, for running multiple
+  tests sequentially in Open Mode for End to End testing. Addresses
+  [#24168](https://github.com/cypress-io/cypress/issues/24168) and
+  [this discussion](https://github.com/cypress-io/cypress/discussions/21628)
+- Added messaging in CI in run mode to direct people to set up recording to the
+  dashboard on failures if they are not already set up to do so. Addresses
```
[the remark this answers is not in the document]
Comment (astone123) [reply content/_changelogs/11.2.0.md]: 1. I agree
2. Yes we did, this should be "Cypress Cloud" rather than "the dashboard"

[the remark this answers is not in the document]
Comment (ryanthemanuel) [reply content/_changelogs/11.2.0.md]: Sounds good @emilyrohrbough and @astone123. Resolved on latest commits.

**Review by ZachJW34 — APPROVED** [review]

File: content/_changelogs/11.2.0.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+## 11.2.0
+
+_Released 11/22/2022_
+
+**Features:**
+
+- Re-implemented the 'Run All Specs' for End to End Testing for running multiple tests sequentially in open mode. This feature had been previously removed in Cypress 10.0. Enable 'Run All Specs' with the new experimental flag, `experimentalRunAllSpecs`. Please leave any feedback around 'Run All Specs'
+  [here](https://github.com/cypress-io/cypress/discussions/21628). Your feedback
+  will help us make product decisions around the future of this feature. Addresses
```
Comment (emilyrohrbough) [inline content/_changelogs/11.2.0.md:9]: oops needs fixed with merged suggestion

File: content/_changelogs/11.2.0.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  tests sequentially in open mode. This feature had been previously removed in
+  Cypress 10.0. Enable 'Run All Specs' with the new experimental flag,
+  `experimentalRunAllSpecs`. Please leave any feedback around 'Run All Specs'
+  [here](https://github.com/cypress-io/cypress/discussions/21628). Your feedback
+  will help us make product decisions around the future of this feature.
+  Addresses [#24168](https://github.com/cypress-io/cypress/issues/24168). tests
+  sequentially in Open Mode for End to End testing. Addresses
+  [#24168](https://github.com/cypress-io/cypress/issues/24168) and
+  [this discussion](https://github.com/cypress-io/cypress/discussions/21628)
+- Added messaging in CI to direct people to set up recording to the Cypress
```
Comment (marktnoonan) [inline content/_changelogs/11.2.0.md:17]: I found this hard to parse, how about:

> Added messaging in CI, after test failures, which directs people to record test results to the Cypress Cloud if they are not already set up to do so. Addresses
  [#24000](https://github.com/cypress-io/cypress/issues/24000)

**Review by marktnoonan — CHANGES_REQUESTED** [review]
We need to fix up the text in the first bullet, might have been a bad merge?

File: content/_changelogs/11.2.0.md (hunk 5 above)
Comment (ryanthemanuel) [reply content/_changelogs/11.2.0.md]: Yeah this was a bad merge. Will be fixed in a sec.

File: content/_changelogs/11.2.0.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+- Re-implemented the 'Run All Specs' for End to End Testing for running multiple
+  tests sequentially in open mode. This feature had been previously removed in
+  Cypress 10.0. Enable 'Run All Specs' with the new experimental flag,
+  `experimentalRunAllSpecs`. Please leave any feedback around 'Run All Specs'
+  [here](https://github.com/cypress-io/cypress/discussions/21628). Your feedback
+  will help us make product decisions around the future of this feature.
+  Addresses [#24168](https://github.com/cypress-io/cypress/issues/24168). tests
+  sequentially in Open Mode for End to End testing. Addresses
+  [#24168](https://github.com/cypress-io/cypress/issues/24168) and
+  [this discussion](https://github.com/cypress-io/cypress/discussions/21628)
```
[the remark this answers is not in the document]
Comment (ryanthemanuel) [reply content/_changelogs/11.2.0.md:16]: Yeah this was a bad merge. Will be fixed in a sec.

File: content/_changelogs/11.2.0.md (hunk 6 above)
Comment (ryanthemanuel) [reply content/_changelogs/11.2.0.md]: I tweaked it a little but I think it's similar to what you have here (and better than what I had previously)

File: content/_changelogs/11.2.0.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+- Added messaging after failures in CI to direct people to set up recording to
+  the Cypress Cloud if they are not already set up to do so. Addresses
+  [#24000](https://github.com/cypress-io/cypress/issues/24000)
+- Visually warn in the Command Log when `cy.session()` fails to restore a
+  session to surface to users potential issues with their setup and validation.
+  Addresses [#24207](https://github.com/cypress-io/cypress/issues/24207)
+
+**Bugfixes:**
+
+- The cy.session() command no longer clears the page between setup and
```
Comment (emilyrohrbough) [inline content/_changelogs/11.2.0.md:23]:
```suggestion
- The [`cy.session()`](/api/commands/session) command no longer clears the page between setup and
```

File: content/_changelogs/11.2.0.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+  tests sequentially in open mode. This feature had been previously removed in
+  Cypress 10.0. Enable 'Run All Specs' with the new experimental flag,
+  `experimentalRunAllSpecs`. Please leave any feedback around 'Run All Specs'
+  [here](https://github.com/cypress-io/cypress/discussions/21628). Your feedback
+  will help us make product decisions around the future of this feature.
+  Addresses [#24168](https://github.com/cypress-io/cypress/issues/24168)
+- Added messaging after failures in CI to direct people to set up recording to
+  the Cypress Cloud if they are not already set up to do so. Addresses
+  [#24000](https://github.com/cypress-io/cypress/issues/24000)
+- Visually warn in the Command Log when `cy.session()` fails to restore a
```
Comment (emilyrohrbough) [inline content/_changelogs/11.2.0.md:17]:
```suggestion
- Visually warn in the Command Log when the [`cy.session()`](/api/commands/session) command fails to restore a
```

File: content/_changelogs/11.2.0.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+- Re-implemented the 'Run All Specs' for End to End Testing for running multiple
+  tests sequentially in open mode. This feature had been previously removed in
+  Cypress 10.0. Enable 'Run All Specs' with the new experimental flag,
+  `experimentalRunAllSpecs`. Please leave any feedback around 'Run All Specs'
+  [here](https://github.com/cypress-io/cypress/discussions/21628). Your feedback
+  will help us make product decisions around the future of this feature.
+  Addresses [#24168](https://github.com/cypress-io/cypress/issues/24168)
+- Added messaging after failures in CI to direct people to set up recording to
+  the Cypress Cloud if they are not already set up to do so. Addresses
+  [#24000](https://github.com/cypress-io/cypress/issues/24000)
```
Comment (emilyrohrbough) [inline content/_changelogs/11.2.0.md:16]:
```suggestion
  [#24000](https://github.com/cypress-io/cypress/issues/24000).
```

File: content/_changelogs/11.2.0.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+- Visually warn in the Command Log when `cy.session()` fails to restore a
+  session to surface to users potential issues with their setup and validation.
+  Addresses [#24207](https://github.com/cypress-io/cypress/issues/24207)
```
Comment (emilyrohrbough) [inline content/_changelogs/11.2.0.md:19]:
```suggestion
  Addresses [#24207](https://github.com/cypress-io/cypress/issues/24207).
```

File: content/_changelogs/11.2.0.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+
+**Bugfixes:**
+
+- The cy.session() command no longer clears the page between setup and
+  validation and will now always clear the page at the end of the command when
+  test isolation is on. Fixes
+  [#21397](https://github.com/cypress-io/cypress/issues/21397)
```
Comment (emilyrohrbough) [inline content/_changelogs/11.2.0.md:26]:
```suggestion
  [#21397](https://github.com/cypress-io/cypress/issues/21397).
```

**Review by emilyrohrbough — APPROVED** [review]

**Review by marktnoonan — APPROVED** [review]
